vertx-db2-client/src/test/java/io/vertx/db2client/DB2ErrorMessageTest.java

```diff
@@ -261,6 +261,24 @@ public void testInsertIntoGeneratedAlwaysColumn(TestContext ctx) {
           }));
       }));
     }
+    
+    @Test
+    public void testDuplicateObject(TestContext ctx) {
+      DB2Connection.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+        conn.query("CREATE TABLE Fortune (\n" + 
+            "  id       integer       NOT NULL GENERATED AS IDENTITY (START WITH 1, INCREMENT BY 1),\n" + 
+            "  message  varchar(2048),\n" + 
+            "  PRIMARY KEY  (id)\n" + 
+            ")").execute(ctx.asyncAssertFailure(err -> {
+            ctx.assertTrue(err instanceof DB2Exception, "The error message returned is of the wrong type.  It should be a DB2Exception, but it was of type " + err.getClass().getSimpleName());
+            DB2Exception ex = (DB2Exception) err;
+            assertContains(ctx, ex.getMessage(), "The object with the name '");
```

Comment:
In Windows, this test (line 275) seems to fail because of the trailing single quote ' (at least removing it made the test pass).
On the Mac, this test passed fine without modification.